src/Nethermind/Nethermind.Api/NethermindApi.cs

```diff
@@ -75,7 +75,7 @@ ILifetimeScope Context
 
         private Dependencies _dependencies;
 
-        public IBlockchainBridge CreateBlockchainBridge()
+        public virtual IBlockchainBridge CreateBlockchainBridge()
```

Comment:
`CreateBlockchainBridge` was broken out to `IBlockchainBridgeFactory` i think. Or no yet? In any case, try not to use `INethermindApi`.

Reply:
Preferably, yes. But its fine for now.